src/shapes/Image.ts

```diff
@@ -718,21 +718,19 @@ export class Image extends FabricObject {
    * @returns {Promise<Image>}
    */
   static fromObject(
-    { filters: f, resizeFilter: rf, src, crossOrigin, ...object }: any,
+    { filters, src, crossOrigin, ...object }: any,
     options: { signal: AbortSignal }
   ): Promise<Image> {
     return Promise.all([
       loadImage(src, { ...options, crossOrigin }),
-      f && enlivenObjects(f, options),
-      rf && enlivenObjects([rf], options),
+      filters ? enlivenObjects<BaseFilter>(filters, options) : undefined,
       enlivenObjectEnlivables(object, options),
-    ]).then(([el, filters = [], [resizeFilter] = [], hydratedProps = {}]) => {
+    ]).then(([el, filters = [], hydratedProps = {}]) => {
       return new this(el, {
         ...object,
         src,
         crossOrigin,
         filters,
-        resizeFilter,
```

Comment:
i already reverted this once, i don't understand where resizeFilters gets enlived now.

Reply:
in enlivenObjectEnlivenables, it is a prop in the map that can be handled by the method as opposed to filters that is an array of stuff to enliven under a prop in the map.